docs/developer-docs/daos/nns/proposal-voting.mdx

```diff
@@ -0,0 +1,174 @@
+* A user with an actively voting neuron who is offering to be followed can register a **known neuron**, also called **named neuron**, in the NNS governance canister. Neurons who would like to follow others can see a list of all registered neurons with their name and optionally a short description and can just select one of them to follow (without having to remember an ID).
+
+## How to become a known neuron
+To become a known neuron in the NNS DAO, you first need to create a neuron and then submit an NNS proposal to register the known neuron in the NNS governance.
+A know neuron has a name and optionally a short description.
+After registering the known neuron, it will be visible to all NNS DAO users when they set up their following.
+To convince the NNS DAO to accept a new known proposal and to provide more context for potential followers, it is
+often useful for known neuron candidates to present themselves to the community already before submitting the proposal.
+
+### Create a neuron - overview
+There are multiple options how to create and set up a neuron that is then registered as a know neuron.
+A known neuron can be operated by an individual person or an organization where one person controls the neuron
+or it can be operated by a group of people.
+Depending on this, different neuron creation options might be more suitable.
+
+### Create a neuron controlled by one party
+
+If you would like to create a known neuron that is operated by one party that knows the neuron's controlling key and can change the neuron, you can use one of the following options:
+* [Create the neuron on the NNS dapp](https://internetcomputer.org/docs/current/developer-docs/daos/nns/nns-app-quickstart#stake-icp-utility-tokens-in-a-neuron).
+* [Create the neuron on the NNS dapp but controlled by a hardware ledger wallet](https://support.dfinity.org/hc/en-us/articles/8760495850900-How-do-I-use-my-Ledger-Nano-with-the-Internet-Computer)
+* [Create the neuron via the quill command line tool](https://support.dfinity.org/hc/en-us/articles/4408752586260-What-is-Quill)
+
+:::info
+If there are multiple people involved a know neuron's voting decision, the known neuron can be set up to follow their individual neurons
+ on the topics that it should vote on.
+Note however that for a neuron which is created this way the **person who created the neuron has the controlling key**.
+This means that this person can modify the neuron and, in particular, can also vote in the name of the neuron without taking the input of others into account.
+This can be suitable for individuals who operate a known neuron or for organizations where one person is trusted with this task.
```

Comment:
```suggestion
However, for a neuron which is created this way, the **person who created the neuron has the controlling key**.
This person can modify the neuron and, in particular, can also vote in the name of the neuron without taking the input of others into account.
This can be suitable for individuals who operate a known neuron, or for organizations where one person is trusted with this task.
```